plugins/discourse-policy/config/settings.yml

```diff
@@ -5,5 +5,13 @@ plugins:
   policy_restrict_to_staff_posts:
     client: true
     default: true
+    hidden: true
```

Comment:
Probably we need a core PR to add this site setting to `deprecated_settings.rb`

Reply:
I have created a TODO for supporting deprecated settings in plugins. 🙏